library/dialog.js

```diff
@@ -187,7 +187,10 @@ function includeScript(url, async, type) {
             console.log('Needed to load:[ ' + url + ' ] For: [ ' + location + ' ]');
             return false;
         }
-
+        
+        if (async === false) {
+            console.log("The url of " + url + " is deprecated due to synchronous requests. Let's find a way to remediate this soon!");
+        }
```

Comment:
@sjpadgett is the dialog.js guru :)

Reply:
@sjpadgett XMLHttpRequests have deprecated synchronous requests and it is expected to go away soon. I think there is only one script that still uses synchronous requests here?